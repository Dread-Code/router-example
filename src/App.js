function App() {
  return (
    <div className="App">
      <h1>App...</h1>
      <hr />
    </div>
  );
}

export default App;
