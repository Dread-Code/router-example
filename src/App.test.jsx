import React from "react";
import { mount } from "enzyme";
import App from "./App";

describe("RouterApp", () => {
  const wrapper = mount(<App />);
  test("should match snapshoot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
