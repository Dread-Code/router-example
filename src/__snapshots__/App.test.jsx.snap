// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RouterApp should match snapshoot 1`] = `
Array [
  <nav
    className="navbar navbar-expand-sm navbar-dark bg-dark"
  >
    <a
      className="navbar-brand"
      href="/"
      onClick={[Function]}
    >
      ReactRouter
    </a>
    <div
      className="collapse navbar-collapse"
    >
      <div
        className="navbar-nav"
      >
        <a
          aria-current="page"
          className="nav-item nav-link active"
          href="/"
          onClick={[Function]}
          style={Object {}}
        >
          Home
        </a>
        <a
          aria-current={null}
          className="nav-item nav-link"
          href="/about"
          onClick={[Function]}
        >
          About
        </a>
        <a
          aria-current={null}
          className="nav-item nav-link"
          href="/login"
          onClick={[Function]}
        >
          Login
        </a>
      </div>
    </div>
  </nav>,
  <div
    className="container"
  >
    <div>
      <h1>
        HomeScreen
      </h1>
      <pre>
        {}
      </pre>
    </div>
  </div>,
]
`;
